bundles/org.openhab.binding.zway/.classpath

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" output="target/classes" path="src/main/java">
+		<attributes>
+			<attribute name="optional" value="true"/>
+			<attribute name="maven.pomderived" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry excluding="**" kind="src" output="target/classes" path="src/main/resources">
+		<attributes>
+			<attribute name="maven.pomderived" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="src" output="target/test-classes" path="src/test/java">
+		<attributes>
+			<attribute name="optional" value="true"/>
+			<attribute name="maven.pomderived" value="true"/>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8">
+		<attributes>
+			<attribute name="maven.pomderived" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.m2e.MAVEN2_CLASSPATH_CONTAINER">
+		<attributes>
+			<attribute name="maven.pomderived" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="output" path="target/classes"/>
```

Comment:
Add the `zway-lib` to the classpath.

Reply:
I do have a question here though. Because I once had forgotten to add the libs in .classpath, and they were automatically added when I opened the project in eclipse and maven ran through. 
So how important is it to add the libs here?

Reply:
It's not a problem for usage. But it will than show up as modified to git and that is probably not desirable. I'll also check other projects as I know I haven't add them to other projects either and make a new PR. 

I've added the zway lib here.

Reply:
Yes, exactly. It does not hurt the build. I do not care much either. But others may wonder why it pops up in the diffs after importing the binding into the IDE. That is the reason to have it inside Git.